projects/packages/search/src/customberg/components/sidebar/tabs.jsx

```diff
@@ -57,7 +57,9 @@ export default function Tabs( { enabledSidebarName, enableSidebar } ) {
  */
 function Tab( { enableSidebar, identifier, label, isActive } ) {
 	// translators: %s: sidebar label e.g: "Options".
-	const ariaLabel = isActive ? sprintf( __( '%s (selected)', 'jetpack-search-pkg' ), label ) : label;
+	const ariaLabel = isActive
+		? sprintf( __( '%s (selected)', 'jetpack-search-pkg' ), label )
```

Comment:
Looks like you'll need to move the translator comment from above the const declaration to right before `__`, so the linter is happy and the extractor picks up the comment.

Reply:
Tried moving the comment below line 60, but the linter was still complaining. So instead, I went ahead and reverted this change since it's irrelevant to this PR.